redis/RedisCluster.php

```diff
@@ -2817,20 +2817,20 @@ public function clearLastError() {}
     /**
      * Get client option
      *
-     * @param   string $name parameter name
+     * @param   string $option parameter name
      *
-     * @return  int     Parameter value.
+     * @return  int|string     Parameter value.
      * @example
      * // return RedisCluster::SERIALIZER_NONE, RedisCluster::SERIALIZER_PHP, or RedisCluster::SERIALIZER_IGBINARY.
      * $redisCluster->getOption(RedisCluster::OPT_SERIALIZER);
      */
-    public function getOption($name) {}
+    public function getOption($option) {}
```

Comment:
Based on https://github.com/phpredis/phpredis/blob/develop/redis_cluster.stub.php#L122  it's `int $option`, right? 
It means that `@param   string $option parameter name` should be `int`.

Reply:
yep. fixed